code/src/components/app-snackbar/AppSnackbar.test.tsx

```diff
@@ -12,7 +13,10 @@ const mockCloseSnackbar = jest.fn();
 
 const mockUseSnackbar = {
   isOpen: true,
-  config: { message: "Test message", variant: "success" },
+  config: {
+    messageTranslationKey: "translation.key",
+    variant: "success"
```

Comment:
Why do you decided to use translation for mock data ?
If you want - ok. Also translation here for `success` to follow one approach
But I don't see a reason to use translations for test. It's mock data for now in your case test
And we use mock texts before

Reply:
As you can see after review, I replaced `message` to `messageTranslationKey` in `openSnackbar` and `openSnackbarWithTimeout` functions. I use translation-like content to emulate real value, that should be passed to this method to avoid confusion. It is just for better code understanding